packages/core/src/metadata.ts

```diff
@@ -90,20 +89,7 @@ class Metadata implements IMetadata {
   }
 
   static fromJSON(value: StringifiedType): Metadata {
-    const {
-      pubKey,
-      polyIDList,
-      generalStore,
-      tkeyStore,
-      scopedStore,
-      nonce,
-      keyType,
-      tssNonces,
-      tssPolyCommits,
-      factorPubs,
-      factorEncs,
-      tssKeyType,
-    } = value;
+    const { pubKey, polyIDList, generalStore, tkeyStore, scopedStore, nonce, keyType, tssNonces, tssPolyCommits, factorPubs, factorEncs } = value;
     const type = keyType in KeyType ? keyType : KeyType.secp256k1;
```

Comment:
```suggestion
    const type = keyType | DEFAULT_KEY_TYPE;
```
and define DEFAULT_KEY_TYPE in common.